src/chttpd/src/chttpd_stats.erl

```diff
@@ -57,16 +68,17 @@ report(HttpReq, HttpResp) ->
     end.
 
 
-report(HttpReq, HttpResp, St) ->
-    case config:get("chttpd", "stats_reporter") of
+report(HttpResp, #st{reporter = Reporter} = St) ->
+    case Reporter of
         undefined ->
```

Comment:
Could pull this into the function head so it would be

```
report(HttpResp, #st{reporter = undefined}) ->
    ok;

report(HttpResp, #st{reporter = Reporter}) ->
    ....
```

Would save an extra case indent level :-)